components/dialpad/sources/new-contact-event/new-contact-event.ts

```diff
@@ -0,0 +1,42 @@
+import { defineSource } from "@pipedream/types";
+import dialpad from "../../app/dialpad.app";
+import constants from "../../common/constants";
+import common from "../../common/common";
+
+export default defineSource({
+  ...common,
+  type: "source",
+  name: "New Contact Event",
+  key: "dialpad-new-contact-event",
+  description: `Emit new contact event subscription. 
+  See [Event doc](https://developers.dialpad.com/reference/webhook_contact_event_subscriptioncreate)
+  and [webhook doc](https://developers.dialpad.com/reference/webhookscreate)`,
+  version: "0.0.1",
+  props: {
+    dialpad,
+    contactType: {
+      type: "string",
+      label: "Contact type",
+      description: "The contact type this event subscription subscribes to.",
+      options: constants.CONTACT_EVENT_TYPE,
+    },
```

Comment:
The `contactType` prop is used in multiple components, so should be put in the app file.
```suggestion
    contactType: {
      propDefinition: [
         dialpad,
         "contactType",
      ],
    },
```